src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -2436,3 +2436,15 @@ def install_kubectl_client():
     except Exception as e:
         telemetry.set_exception(exception=e, fault_type=consts.Download_And_Install_Kubectl_Fault_Type, summary="Failed to download and install kubectl")
         raise CLIInternalError("Unable to install kubectl. Error: ", str(e))
+
+def get_issuer_url():
+    # Get the Issuer Url from the signingkey CR in the cluster specified
+    kubectl_client_location = install_kubectl_client()
+
+    kubectl_cmd = [kubectl_client_location, "get", "signingkeys.clusterconfig.azure.com", "-n", "azure-arc", "signingkey", "-o=jsonpath={.status.clusterIssuerUrl}"]
```

Comment:
please use k8s python sdk of kubectl to make these calls whenever possible. We fall back to using kubectl calls only when relevant sdk method is not available

Reply:
example - https://github.com/Azure/azure-cli-extensions/blob/6b0c3466e313e0d7ea7d331c673275d15d64eff5/src/connectedk8s/azext_connectedk8s/custom.py#L284